src/styles/main.scss

```diff
@@ -1,3 +1,81 @@
-body {
+$cell-size: 100px;
+$gap-size: 1px;
+$padding: 10px;
+$sheet-with: $cell-size * 7 + $gap-size * 6;
```

Comment:
I would create a new scss file and put all the variables there. 

You may reference to checklist (p.5)

Reply:
➕ , create a separate file for variables. Also, avoid magic numbers